src/index.ts

```diff
@@ -15,13 +16,30 @@ export const defaultLanguageSelector = ['css', 'javascript', 'html', 'mdx', 'typ
 
 export { tailwindcssData } from './cssData';
 
+function parseModelLike(value: ModelLike): ChangedContent {
+  const modelOrString = Uri.isUri(value) ? editor.getModel(value) : value;
+  if (!modelOrString) {
+    return { content: '' };
+  }
+  if (typeof modelOrString === 'string') {
+    return { content: modelOrString };
+  }
+  return {
+    content: modelOrString.getValue(),
+    extension: modelOrString.getLanguageId(),
+  };
+}
+
 export const configureMonacoTailwindcss: typeof import('monaco-tailwindcss').configureMonacoTailwindcss =
   ({ tailwindConfig, languageSelector = defaultLanguageSelector } = {}) => {
-    const workerManager = createWorkerManager<TailwindcssWorker, MonacoTailwindcssOptions>({
-      label: 'tailwindcss',
-      moduleId: 'monaco-tailwindcss/tailwindcss.worker',
-      createData: { tailwindConfig },
-    });
+    const workerManager = createWorkerManager<TailwindcssWorker, MonacoTailwindcssOptions>(
+      { editor },
```

Comment:
i guess an inbetween step for https://github.com/remcohaszing/monaco-tailwindcss/issues/12